src/renderer/components/Wallet/WalletHeading/WalletHeading.vue

```diff
@@ -39,7 +39,7 @@ export default {
 
   watch: {
     currentWallet () {
-      if (this.activeWallet && this.activeWallet.id !== this.currentWallet.id) {
+      if (this.activeWallet !== this.currentWallet.id) {
```

Comment:
You should add `&& this.activeWallet.id` again, as it currently doesn't check against an id anymore

Reply:
cool, on it!